zbus/src/fdo.rs

```diff
@@ -272,7 +272,7 @@ macro_rules! gen_object_manager_proxy {
 gen_object_manager_proxy!(true, false);
 assert_impl_all!(ObjectManagerProxy<'_>: Send, Sync, Unpin);
 
-/// Service-side [Object Manager][om] interface implementation.
+/// Server-side [Object Manager][om] interface implementation.
```

Comment:
This was very intentional. The "server" in D-Bus is typically the broker and most services are just clients of this server.

Reply:
Right, I wasn't sure, but line 357 says "Server-side" and seems to be talking about the same side, so I picked one to be consistent.

Should line 357 also be changed to "Service-side"? Or is it actually talking about building a broker?

Reply:
Right, so that's our fault. Until recently we were using `server` to mean a service and it was confusing as hell. I recently fixed that in most places, but obviously I didn't do a very thorough job. :cry: 

> Should line 357 also be changed to "Service-side"? Or is it actually talking about building a broker?

Yeah, that's a left-over from my changes. It should be `Service-side`. There are certainly spots where we really are talking about the server-side but they're likely guarded by the `bus-impl` feature so it should be easy to tell those ones apart.

Reply:
Fixed on lines 45, 112, and 357.